src/service/firebase.js

```diff
@@ -14,4 +15,8 @@ const firebaseConfig = {
 // Initialize Firebase
 const instance = firebase.initializeApp(firebaseConfig);
 
-export const auth = instance.auth()
\ No newline at end of file
+export const auth = instance.auth()
+
+const db = firebase.firestore()
+
+export const addItem = (collection, item) => db.collection(collection).add(item)
```

Comment:
I know you are not a fan of comments, but for the firebase services create a document or comment some parts of the code explaining each function and how to duplicate it for other values that will be added to the db

Reply:
That is true I will add a task for a knowledge base section on github.